installer/bundle/bundle_skel.sh

```diff
@@ -817,6 +834,21 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+            for i in bundles/*-bundle-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+
+                # It's possible we have a test file without bundle; if so, ignore it
+                BUNDLE=`basename $i -bundle-test.sh`
```

Comment:
I still do not understand the meaning of this comment" It's possible we have a test file without bundle; if so, ignore it " , can you rephrase it if the test file is about the bundle-test.sh itself and not some other test input data?